x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_source_logic.test.ts

```diff
@@ -93,75 +93,124 @@ describe('AddSourceLogic', () => {
     expect(AddSourceLogic.values).toEqual(defaultValues);
   });
 
+  const setupExpectedState = ({ setSourceConfigData = {} } = {}) => {
+    const sourceConfigDataSettings = Object.keys(setSourceConfigData).length > 0
+      ? {
+          dataLoading: false,
+          sourceConfigData: setSourceConfigData,
+        }
+      : {};
+
+    return {
+      ...defaultValues,
+      ...sourceConfigDataSettings,
+    };
+  };
   describe('actions', () => {
     it('setSourceConfigData', () => {
       AddSourceLogic.actions.setSourceConfigData(sourceConfigData);
-
-      expect(AddSourceLogic.values.sourceConfigData).toEqual(sourceConfigData);
-      expect(AddSourceLogic.values.dataLoading).toEqual(false);
-      expect(AddSourceLogic.values.buttonLoading).toEqual(false);
-      expect(AddSourceLogic.values.clientIdValue).toEqual(
-        sourceConfigData.configuredFields.clientId
-      );
-      expect(AddSourceLogic.values.clientSecretValue).toEqual(
-        sourceConfigData.configuredFields.clientSecret
-      );
-      expect(AddSourceLogic.values.baseUrlValue).toEqual(sourceConfigData.configuredFields.baseUrl);
+      const EXPECTED_BASE_STATE = setupExpectedState();
```

Comment:
Why did you call it `EXPECTED_STATE_BASE` in previous commit but `EXPECTED_BASE_STATE` here?